airbyte-integrations/connectors/source-fastbill/acceptance-test-config.yml

```diff
@@ -3,9 +3,9 @@
 # Airbyte doesn't have a Fastbill test account to run tests
 connector_image: airbyte/source-fastbill:dev
 acceptance_tests:
-  # spec:
-  #   tests:
-  #     - spec_path: "source_fastbill/spec.yaml"
+  spec:
+    tests:
+      - spec_path: "manifest.yaml"
   connection:
     tests:
       #     - config_path: "secrets/config.json"
```

Comment:
I created a new sandbox trial account for the connector. Let's go ahead and uncomment these tests

Reply:
Cool!